docs/build/references/platform-transaction-serialization.md

```diff
@@ -891,6 +891,163 @@ Let’s make an unsigned add delegator tx that uses the inputs and outputs from
+
+An unsigned create chain tx contains a `BaseTx`, `SubnetID`, `ChainName`, `VMID`, `FxIDs`, `GenesisData` and `SubnetAuth`. The `TypeID` for this type is `0x0000000f`.
+
+* **`BaseTx`**
+* **`SubnetID`** ID of the Subnet that validates this blockchain
+* **`ChainName`** A human readable name for the chain; need not be unique
+* **`VMID`** ID of the VM running on the new chain
+* **`FxIDs`** IDs of the feature extensions running on the new chain
+* **`GenesisData`** Byte representation of genesis state of the new chain
+* **`SubnetAuth`** Auth that will be allowing this validator into the network
+
+### **Gantt Unsigned Create Chain Tx Specification**
+
+```text
++--------------+-------------+-----------------------------------------+
+| base_tx      : BaseTx      |                     size(base_tx) bytes |
++--------------+-------------+-----------------------------------------+
+| subnet_id    : SubnetID    |                                32 bytes |
++--------------+-------------+-----------------------------------------+
+| chain_name   : ChainName   |               2 + len(chain_name) bytes |
++--------------+-------------+-----------------------------------------+
+| vm_id        : VMID        |                                32 bytes |
++--------------+-------------+-----------------------------------------+
+| fx_ids       : FxIDs       |                   4 + size(fxIDs) bytes |
++--------------+-------------+-----------------------------------------+
+| genesis_data : GenesisData |             4 + size(genesisData) bytes |
++--------------+-------------+-----------------------------------------+
+| subnet_auth  : SubnetAuth  |                  size(subnetAuth) bytes |
++--------------+-------------+-----------------------------------------+
+               | 74 + size(base_tx) + size(chain_name) + size(fxIDs) + |
+               |            size(genesisData) + size(subnetAuth) bytes |
++--------------+-------------------------------------------------------+
+```
+
+### **Proto Unsigned Create Chain Tx Specification**
+
+```text
+message CreateChainTx {
+    BaseTx base_tx = 1;               // size(base_tx)
+    SubnetID subnet_id = 2;           // 32 bytes
+    ChainName chain_name = 3;         // 2 + len(chain_name) bytes
+    VMID vm_id = 4;                   // 32 bytes
+    FxIDs fx_ids = 5;                 // 4 + size(fxIDs) bytes
+    GenesisData genesis_data = 6      // 4 + size(genesisData) bytes
+    SubnetAuth subnet_auth = 7;       // 8 + size(subnetAuth) bytes
+}
+```
+
+### **Unsigned Create Chain Tx Example**
+
+Let’s make an unsigned create chain tx that uses the inputs and outputs from the previous examples:
+
+* **`BaseTx`**: `"Example BaseTx as defined above with ID set to 0f"`
+* **`SubnetID`**: `24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1`
+* **`ChainName`**: `EPIC AVM`
+* **`VMID`**: `avm`
+* **`FxIDs`**: `0x736563703235366b316678000000000000000000000000000000000000000000`
```

Comment:
I feel like `VMID` and `FxIDs` should either both be human readable here - or both hex. Also `FxIDs` should be an array right?

Reply:
Good call. Resolved.